src/deadline/job_attachments/_glob.py

```diff
@@ -0,0 +1,70 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+import json
+from pathlib import Path
+from typing import List, Optional
+from deadline.client.exceptions import NonValidInputError
+from deadline.job_attachments.models import GlobConfig
+
+
+def _process_glob_inputs(glob_arg_input: str) -> GlobConfig:
+    """
+    Helper function to process glob inputs.
+    glob_input: String, can represent a json, filepath, or general include glob syntax.
```

Comment:
Is the plan here to accept JSON as a CLI parameter input? If so, maybe we can adjust that to accept multiple arguments of include & exclude instead? I believe CLI interfaces that don't use JSON that way are generally more comfortable to use.

Reply:
Do you mean something like 
```--include abc/*.xyz,def/*.123```. ?

I was going by the OpenJD suggestion for globs, but happy to go about it either ways.

Reply:
I was thinking of something like this example: https://github.com/aws-deadline/deadline-cloud/pull/446/files#diff-33ab9cc270d578c6903c5e1cf7eb1e25e518cfc7222717f4599df29a368d0cc5R8

Reply:
For flexibility, I can also add `--include` and `--exclude` as comma separated inputs w/ the CLI version too. This function her specifically supports JSON file and JSON string.

Reply:
We chatted internally within the team. This code provides support for a JSON based glob syntax and can be used for large lists of include and exclude. For CLI use cases, I will also add `--include` and `--exclude` so both basis are covered.